core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceTypeUtilTest.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2018 LINE Corporation
```

Comment:
```suggestion
 * Copyright 2020 LINE Corporation
```

Reply:
Thanks. Fixed for this file.

This file is obviously created 2020, but how about the other file? Should I replace the line in 
`.../src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceTypeUtil.java` with 

```
Copyright 2018-2020 LINE Coporation
```

Reply:
Other files don't seem to need to be changed. 😄